repo.toml

```diff
@@ -153,8 +153,8 @@ output_format = "linkcheck"
 docs_root = "${root}/gpu-operator"
 project = "gpu-operator"
 name = "NVIDIA GPU Operator"
-version = "24.9.2"
-source_substitutions = { version = "v24.9.2", recommended = "570.86.15" }
+version = "25.3.0"
+source_substitutions = { version = "v25.3.0", recommended = "570.86.15" }
```

Comment:
I believe the `recommended` field here is inaccurate. `570.124.06` is the default and recommended driver version to 25.3.0.

Reply:
updated